pages/js/frameworks.md

```diff
@@ -4,6 +4,14 @@ permalink: /javascript/frameworks/
 sidenav: js
 ---
 # Frameworks
+
+**We recommend using [React]() for projects when possible.** As a team, we enjoy using React because it has a strong component model, enables easy server side rendering, and faciliates debugging by making the UI elements a function of the application state. React is most akin to the "view layer" in an MVC (Model-View-Controller) architecture and is frequently used with some other libraries. In our projects, we tend to use:
+
+* [Redux](https://www.npmjs.com/package/redux) for managing state
+* [Webpack](https://www.npmjs.com/package/webpack) and [Babel](https://babeljs.io/) for compiling our Javascript
+
+However, we are not dogmatic about React - we just vastly prefer it. Here are some other popular frameworks that have been used at 18F and some of their key characteristics.
+
```

Comment:
Brought this over from the ancient PR that never got merged, just to have something to work with.

Reply:
I still think, especially given people's concerns about Facebook, that we should say "React or React-like" frameworks. 

I'd also feel comfortable being more dogmatic about it, and say "Use a react-like framework unless you have a compelling reason not to, such as the partner already using another framework". I just think the ramp-up time for developers diving into one person's fave framework from six months ago is much greater than any benefit from that framework in most cases.

Unless we have a strong second choice, I'd be happy dropping the angular and Backbone sections, too.